app/helpers/application_helper.rb

```diff
@@ -4,9 +4,7 @@ def field_error(resource, key)
   end
 
   def active_tab(identifier)
-    if (request.path == root_path) && (request.path == identifier)
-      "active"
-    elsif request.path.include?(identifier) && identifier != root_path
+    if request.path.include?(identifier) && identifier != root_path
```

Comment:
I think the latter condition also needs removing here, as this seems to be a defence against the identifier being `/` and marking all tabs as active.

If we're assuming the identifier is never `/`, then we can remove this too
```suggestion
    if request.path.include?(identifier)
```